packages/dd-trace/src/platform/node/index.js

```diff
@@ -57,3 +67,7 @@ const platform = {
 process.once('beforeExit', () => emitter.emit('exit'))
 
 module.exports = platform
+
+function hasSupportedAsyncLocalStorage () {
+  return semver.satisfies(process.versions.node, '^14.5 || ^12.18.5 || >14')
```

Comment:
Shouldn't this be `>=14.5 || ^12.18.3`?

Reply:
Turns out it was actually `>=14.5 || ^12.19.0`. Node changelog is not the easiest thing to read. Also not sure why the condition for `>14` is needed.

Reply:
The `>14` was necessary due to the use of the caret in `^14.5`. This is now fixed, as well as 12.19.0.